Draft.md

```diff
@@ -168,159 +168,201 @@ Im folgenden stellen wir einen kommentierten Fragenkatalog mit Kriterien zur Beu
+* **Gibt es eine Entwicklerdokumentation, welche eine evtl. Softwareerweiterung fördert?** Eine Entwicklerdokumentation bietet einem Entwickler den einfachsten Einstieg in der Beschäftigung mit der Erweiterung oder Nutzung einer Software. Eine aussagekräftige *README*-Datei, die kurz die Verwendung des Programmes mit den Standardeinstellungen demonstriert, ist eine Mindestanforderung. Ein Beispiel dafür ist  [Bibtex_JS](https://github.com/pcooksey/bibtex-js). Idealerweise werden zusätzlich Beispieldaten für ein besseres Verständnis des Programmablaufs beigelegt und ggf. weitere häufig verwendete Anwendungsfälle der Software in Beispielen vorgestellt. Für das gerade genannte Beispiel trifft dies zu: [Bibtex_JS Beispiele](https://github.com/pcooksey/bibtex-js/tree/master/test)). Abhängig von der Komplexität der Software kann es sinnvoll sein, ein ggf. auch von einer Nutzercommunity gepflegtes Wiki bereitzustellen, um fortgeschrittene Optionen zu erläutern (siehe [Bibtex_JS Wiki](https://github.com/pcooksey/bibtex-js/wiki)).
 
-* **Hat der Quellcode Tests, die die Kernfunktionen des Programms testen und diese für andere Entwickler aufzeigen?** Jede Software kann durch Tests auf ihre antizipierte Funktionalität hin überprüft werden. Tests erfüllen hierbei zwei grundlegende Funktionen. Zum einen zeigen sie auf welche Anwendungsfälle hin die Entwickler die Software antizipiert haben und geben diese Information in Form von Tests weiter. Zum anderen geben Tests Auskunft darüber, welche Funktionen im jeweiligen Programm als stabil, fehleranfällig oder ggf. noch optimierungsbedürftig gelten können. Kombiniert mit einem *Continuous Integration* Prozess können Testauswertungen mit jeder Änderung am Quellcode durch die Entwickler erstellt werden. Als *Continuous Integration* Prozess bezeichnet man eine Reihe von Aufgaben welche mit dem Quellcode in einem Repository durchgeführt wird. Jede der Aufgaben erzeugt ein Ergebnis welches einen Mehrwert für das Projekt bietet. Beispielsweise kann ein Releasefile (.exe), eine Quellcode Dokumentation oder auch wie in diesem Fall ein Testreport mit einem *Continuous Integration* Prozess erzeugt werden. Ein solcher Prozess wird üblicherweise nach jeder Änderung des Quellcodes erneut ausgeführt um die Ergebnisse der Aufgaben immer so aktuell wie möglich zu halten.
+> [MT] Hm, jetzt wo ich die Beschreibung zu 'Entwicklerdokumentation' gelesen habe, habe ich den Eindruck, dass das eine ganz normale Nutzerdoku meint; Das wurde ja oben unter Hilfefunktionen, Tutorials und Community schon beschrieben und überlappt sich teilweise. Da könnte man entweder genauer unterscheiden oder auch streichen.
```

Comment:
Jein. Der Schwerpunkt liegt im Gegensatz zur Nutzerdoku hier auf der Wiederverwendbarkeit der Software für den Programmierer. Also die Intention ist "Hier kannst du starten wenn du die Software erweitern willst und so fängst du an" wohingegen die Nutzerdoku für mich ist "So kannst du die Software bedienen und wenn du eine Anwendung hast die mit der Software sprechen soll dann sollte sie diese so und so ansprechen wenn sie läuft (API)"

Reply:
Ja, den Unterschied sehe ich auch so. Allerdings kann ich den im Text, bzw. dann in dem genannten Beispiel nicht erkennen.